### PR TITLE
Summary Bar or Pie chart

### PR DESCRIPTION
**Pull request by shadrackkibet**
@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

**Comment by lilyclements** [comment]
Reviewing this now

**Comment by lilyclements** [comment]
Have you tried to run this before? I know I haven't, so this was my first run of it.

I did column statistics on my dodoma dataset (Rain by Year, calculation type as sum)
I then tried to use this dialog to have my Y as my `sum_Rain` and my Factor as `Year`. This gave an immediate error which I think is to do with using the parameter name as `y` for `y = sum_Rain`? I'm not entirely sure as I don't know what it's meant to run instead! However, I'm hoping the issue is just this parameter name. In R, it gives the following error
> Error: stat_count() must not be used with a y aesthetic.

Have I misused this dialog? How did you get it to work when you tested it?

**Comment by shadrackkibet** [comment]
@lilyclements  i am not getting this error here. perhaps you could demonstrate this via skype?

File: instat/dlgSummaryBarOrPieChart.vb [hunk 1]
```diff
@@ -231,7 +232,7 @@ Public Class dlgSummaryBarOrPieChart
         SetDialogOptions()
     End Sub
 
-    Private Sub CoreControls_ContenctsChanged() Handles ucrYReceiver.ControlContentsChanged, ucrFactorReceiver.ControlContentsChanged, ucrSaveSummaryBar.ControlContentsChanged
+    Private Sub CoreControls_ContenctsChanged() Handles ucrReceiverYVariable.ControlContentsChanged, ucrReceiverFactor.ControlContentsChanged, ucrSaveSummaryBar.ControlContentsChanged
```
Comment (lilyclements) [inline instat/dlgSummaryBarOrPieChart.vb]: Can this read
`CoreControls_ContentsChanged()`

Comment (shadrackkibet) [reply instat/dlgSummaryBarOrPieChart.vb]: good spot! will change this.

**Comment by lilyclements** [comment]
@shadrackkibet have you and @maxwellfundi discussed the error?

**Comment by shadrackkibet** [comment]
This has conflicting files. I will close this and move changes to another branch.

**Pull request closed without merging**